gamestonk_terminal/cryptocurrency/cryptocurrency_helpers.py

```diff
@@ -1048,13 +1048,10 @@ def plot_order_book(
     ax.plot(asks[:, 0], asks[:, 2], "r", label="asks")
     ax.fill_between(asks[:, 0], asks[:, 2], color="r", alpha=0.4)
 
-    # plt.grid(b=True, which="major", color="#666666", linestyle="-")
-    # plt.minorticks_on()
-    # plt.grid(b=True, which="minor", color="#999999", linestyle="-", alpha=0.2)
-    plt.legend(loc=0)
-    plt.xlabel("Price")
-    plt.ylabel("Size (Coins) ")
-    plt.title(f"Order Book for {coin}")
+    ax.legend(loc=0)
```

Comment:
Is there a text string we can use instead of a 0? Is it "top left" or something similar

Reply:
I changed to loc="best"